src/System.Private.ServiceModel/tests/Common/Infrastructure/testproperties.props

```diff
@@ -150,6 +150,10 @@
     <PropertyGroup Condition="'$(SSL_Available)' == ''">
       <SSL_Available></SSL_Available>
     </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IncludeTestsWithIssues)' == '' and $(WithCategories.ToLower().Contains('failing'))">
+    <IncludeTestsWithIssues>all</IncludeTestsWithIssues>
```

Comment:
NIT: I personally feel `true` is more intuitive than `all` (if I just guess what value I can set).